linter_exclusions.yml

```diff
@@ -2832,6 +2832,28 @@ stack-hci cluster create:
     aad_service_principal_object_id:
       rule_exclusions:
         - option_length_too_long
+stack-hci-vm network nsg rule create:
```

Comment:
These are the same options used by NetworkSecurityGroup in regular Azure. We want to match them as much as possible since we are providing the same API for hybrid workloads running on-prem.